### ISSUE TITLE
Add Templates

### ISSUE THREAD

**Issue by pkoo8** [pkoo8/ResumeBuilder#2]
Add some Templates for reference to build a resume(at least 5 valid formats) in latex to get a 'pr-accepted' or 'hacktoberfest-accepted' label Multiple PR's will be accepted

**Comment by pavan-nambi** [comment]
can i try?

**Comment by pkoo8** [comment]
Yes @pavan-nambi you have to add 5 valid templates along with the role of the positions they will be valid for
